app/views/course/achievements/_form.html.slim

```diff
@@ -27,7 +27,7 @@
           th = t('course.condition.type.header')
           th = t('course.condition.description.header')
       tbody
-        - @achievement.conditions.each do |condition|
+        - @achievement.conditions.includes(:conditional).each do |condition|
```

Comment:
Yes. I'll pass the conditional as a local then. Those partials may be used by other conditionals in future.